drivers/amazonec2/amazonec2.go

```diff
@@ -1107,14 +1124,14 @@ func (d *Driver) configureSecurityGroupPermissions(group *ec2.SecurityGroup) ([]
 
 	perms := []*ec2.IpPermission{}
 
-	if !hasPorts["22/tcp"] {
-		perms = append(perms, &ec2.IpPermission{
-			IpProtocol: aws.String("tcp"),
-			FromPort:   aws.Int64(22),
-			ToPort:     aws.Int64(22),
-			IpRanges:   []*ec2.IpRange{{CidrIp: aws.String(ipRange)}},
-		})
-	}
+        if !hasPorts[fmt.Sprintf("%d/tcp", d.BaseDriver.SSHPort)] {
+                perms = append(perms, &ec2.IpPermission{
+                        IpProtocol: aws.String("tcp"),
+                        FromPort:   aws.Int64(22),
```

Comment:
As noted by @axw-pivorra, this should be `aws.Int64(int64(d.BaseDriver.SSHPort)),`

Reply:
As it's port range, FromPort and ToPort needs to be set as aws.Int64(int64(d.BaseDriver.SSHPort))
Correction to apply on line 1130